pkg/appconsts/testground/app_consts.go

```diff
@@ -0,0 +1,7 @@
+package testground
+
+const (
+	Version              uint64 = 420420420
+	SquareSizeUpperBound int    = 512
+	SubtreeRootThreshold int    = 64
```

Comment:
[question] related to https://github.com/celestiaorg/celestia-app/pull/2456/files#r1417502424

why define this constant if it is never used?